.github/workflows/main.yml

```diff
@@ -52,6 +52,9 @@ jobs:
         if: ${{ matrix.lint }}
         run: mix format --check-formatted
 
+      - name: Run Dialyzer
+        run: mix dialyzer
```

Comment:
I'm ok with this, but we need to cache the PLTs built by Dialyzer. Something like [this](https://github.com/whatyouhide/redix/blob/3ba88b96e08ceab95e0839fc48f58adf488709dd/.github/workflows/main.yml#L56-L68) 🙃

Reply:
Surely :) I just didn't wanna go full-in until I understood if there was a chance to accept the change. Will take care. Thanks.

Reply:
c342a98